CONTRIBUTING.standards.md

```diff
@@ -0,0 +1,110 @@
+# FDC3 Charter
+
+This Project Charter establishes the terms under which this FINOS Project will operate. By Joining the Project or any Working Group, each Participant agrees to be bound by the terms of this Charter and the policies listed in Section 1.2.
+
+1\.    Establishment.
+
+1.1.    Purposes. The purposes of this Project are: To develop specific protocols and taxonomies to advance the ability of desktop applications in financial workflows to interoperate in a plug-and-play fashion, without prior bi-lateral agreements.
+
+1.2.    Initial Steering Members. The initial Steering Members of this Project are: OpenFin, J.P. Morgan.
```

Comment:
@nkolba @mindthegab I am not sure who the initial steering members, or indeed the current steering members, of FDC3 are, but given that there is an initial requirement of at least two steering members, I went for OpenFin and J.P. Morgan, as they were two of the earliest involved members of FDC3 that I am aware of.

Happy to amend this as required, just let me know who to put down.

Reply:
@rikoe if this is meant to indicate the initial makeup of the PMC, it would be: OpenFin, JPM, FactSet, Citadel, Refinitiv, and IHS Markit.

Reply:
@nkolba - updated